riscv-c-api.md

```diff
@@ -170,6 +170,90 @@ For example:
 * `__attribute__((interrupt("supervisor")))`
 * `__attribute__((interrupt("machine")))`
 
+### `__attribute__((target("<ATTR-STRING>")))`
+
+`target` attribute used for enable or disable a set of features or extensions
+for the fucntion.
+
+For instance, you can enable the `v` extension for a specific function even if
+`-march` or `-mcpu` does not include the `v` extension. Additionally,
+this won't alter the global setting.
+```c
+__attribute__((target("+v")))
```

Comment:
```suggestion
__attribute__((target("arch=+v")))
```
Is this what was intended?

Reply:
Oh, you catch a bug, it's not intentionally